binaries/ofs.uio/uio/ofs_uio.py

```diff
@@ -70,6 +72,48 @@ def verify_pcie_address(pcie_address):
     return True
 
 
+def verify_hex(str):
+    regx = re.compile("0[xX][0-9a-fA-F]+")
```

Comment:
You don't need to compile the regex each time you call the function.  Do it globally, once, and that will be sufficient.

Reply:
Fixed